### DISCUSSION TITLE
Path or bread-crumbs Display for nested/recursive relations

### DISCUSSION THREAD

**Discussion by gerolds** [directus/directus#16341] [Feature Requests]
We are using nested/recursive/self-referencing m2o relations to build a tree structures which display their children with a Tree View.

Typically we have these fields in our collections:

```
tree_node<<collection>> {
    + children:tree_view -> tree_node.parent;
    + parent:m2o -> tree_node.id;
}
```

Now when looking at these collections in a table view or when selecting parents from a list it is quite inconvenient to find the right one because there is no Display for m2o relations that understands recursion. So we end up with a display template like this

```
{"template":".. / {{parent.parent.name}} / {{parent.name}} / {{name}}"}
```

This is quite inefficient and slows down the admin UI. It would be neat if we had a Path/Breadcrumb-Display  (in place of the Raw-/Related-Values Display) we could assign to the parent field definition that solves this procedurally without having to assign such cumbersome template strings.

**Comment by Clintonkes** [comment]
What is the email about?

**Reply by br41nslug** [reply]
Some people are spamming github repo's tagging random people.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
